addons/account/views/account_move_views.xml

```diff
@@ -1035,24 +1045,26 @@
                                         </sheet>
                                     </form>
                                 </field>
-                                <group col="4">
-                                    <group colspan="3">
+                                <group col="12" class="oe_invoice_lines_tab">
```

Comment:
if the client don't update its `account` module, is this causing an issue?

Reply:
without update (using the french translation)
![image](https://user-images.githubusercontent.com/60702485/166202830-d68e231e-8f42-49d9-a12c-29f8efbd3a81.png)
After update
![image](https://user-images.githubusercontent.com/60702485/166203190-7f076cb2-2ce2-4956-8ecb-268e140b75fa.png)
So the only issue is the translations: not happening for the new phrases in the pot file, but I guess it's normal ?

Reply:
ok then :)